qiskit/qasm3/exporter.py

```diff
@@ -325,6 +336,14 @@ def __init__(
         self._circuit_ctx = []
         self.push_context(quantumcircuit)
         self.includeslist = includeslist
+        # `_global_io_declarations` and `_global_classical_declarations` are stateful, and any
+        # operation that needs a parameter can append to them during the build.  We make all
+        # classical declarations global because the IBM QSS stack (our primary consumer of OQ3
```

Comment:
We can hope for the future :)
```suggestion
        # classical declarations global because the IBM QSS stack (our initial primary consumer of OQ3
```

Reply:
Done in 13b75a894.